data/templates/src/elf/dll/elf_dll_x64_template.s

```diff
@@ -50,7 +50,7 @@ shdr:
   dd    0                          ; sh_link
   dd    0                          ; sh_info
   dq    8                          ; sh_addralign
-  dq    dynsz                      ; sh_entsize
+  dq    7                          ; sh_entsize
```

Comment:
This was changed to a 7 as `sh_entsize` is the number of entries in a table, not the actual memory size in bytes.